test/swivel/Swivel.sol

```diff
@@ -272,17 +272,17 @@ contract Swivel {
     
     filled[hash] += a;
         
-    uint256 premiumFilled = (((a * 1e18) / o.principal) * o.premium) / 1e18;
-    uint256 fee = ((premiumFilled * 1e18) / fenominator[3]) / 1e18;
+    uint256 premiumFilled = (a * o.premium) / o.principal;
 
     Erc20 uToken = Erc20(o.underlying);
-    // transfer premium minus fee from maker to sender
-    uToken.transferFrom(o.maker, msg.sender, premiumFilled - fee);
+    // transfer premium from maker to sender
+    uToken.transferFrom(o.maker, msg.sender, premiumFilled);
 
+    uint256 fee = ((premiumFilled * 1e18) / fenominator[3]) / 1e18;
```

Comment:
I'll move this up to 276 just for code readability/consistency sake later on

Reply:
looks back at other methods..

yeah, its dumb everywhere now. we really shouldn't spend the resources calculating and storing a variable that isn't immediately used and may not be used ah all (if the `transferFrom` after it fails, this var was never needed to begin with)